lib/msal-common/src/telemetry/performance/IPerformanceClient.ts

```diff
@@ -27,6 +27,6 @@ export interface IPerformanceClient {
     removePerformanceCallback(callbackId: string): boolean;
     addPerformanceCallback(callback: PerformanceCallbackFunction): string;
     emitEvents(events: PerformanceEvent[], correlationId: string): void;
-    startPerformanceMeasurement(measureName: string, correlationId: string): IPerformanceMeasurement;
+    startPerformanceMeasuremeant(measureName: string, correlationId: string): IPerformanceMeasurement;
```

Comment:
This will be breaking going back as well (for people on the current, latest browser version). Should we perhaps include both here and then remove the wrong one with a major bump?

Reply:
Sounds good. I can open another PR for that bump.